### PR TITLE
Fixes #29528 - rubocop task definition

### PR DESCRIPTION
**Pull request by ezr-ondrej**

### CODE REVIEW

File: lib/tasks/foreman_ansible_tasks.rake [hunk 1]
```diff
@@ -13,19 +13,15 @@ namespace :test do
...
-                         "#{ForemanAnsible::Engine.root}/test/**/*.rb"]
-      end
-    rescue StandardError
-      puts 'Rubocop not loaded.'
+  begin
+    require 'rubocop/rake_task'
+    RuboCop::RakeTask.new(:rubocop) do |task|
+      task.patterns = ["#{ForemanAnsible::Engine.root}/app/**/*.rb",
+                       "#{ForemanAnsible::Engine.root}/lib/**/*.rb",
+                       "#{ForemanAnsible::Engine.root}/test/**/*.rb"]
```
Comment (ezr-ondrej) [inline lib/tasks/foreman_ansible_tasks.rake:21]: shouldn't this be `Include: ...` in `.rubocop.yml`?

Comment (xprazak2) [reply lib/tasks/foreman_ansible_tasks.rake:21]: It might be worth moving it to `.rubocop.yml` so that we are consistent across plugins...

Comment (ezr-ondrej) [reply lib/tasks/foreman_ansible_tasks.rake:21]: I've tried that, but it seems, that it doesn't work for the rake task actually :disappointed:

**Comment by xprazak2** [comment]
codeclimate is failing, should be an easy fix

**Comment by ezr-ondrej** [comment]
> codeclimate is failing, should be an easy fix - funny thing is that rubocop on jenkins is passing...

This was introduced in ruby 2.5. I didn't want to go 2.4 incompatible just to make codeclimate happy :thinking: but I guess with Rails 6 now we are ok to do that :)

**Comment by xprazak2** [comment]
Thanks!

**Pull request merged**